### PR TITLE
Update user guide for sort command

### LINKED ISSUE #109: Update user guide for sort command

### PR DESCRIPTION
**Pull request by DesSnowy**
closes #109

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -246,7 +260,7 @@ AddressBook data are saved in the hard disk automatically after any command that
 
 AddressBook data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
-<box type="warning" seamless>
+<box type="warning" seamless
```
Comment (SwaminathanViswa) [inline docs/UserGuide.md:263]: Is the `>` intended to be missing?

**Review by SwaminathanViswa — COMMENTED** [review]
Apart from that comment, the rest looks good to me!

File: docs/UserGuide.md (hunk 1 above)
Comment (DesSnowy) [reply docs/UserGuide.md:263]: Oh I think it got accidentally removed. Will be fixing it.

**Review by SwaminathanViswa — APPROVED** [review]
LGTM!

**Pull request merged**